pkgs/by-name/gh/ghciwatch/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, nix-update-script
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ghciwatch";
+  version = "0.5.10";
+
+  src = fetchFromGitHub {
+    owner = "MercuryTechnologies";
+    repo = "ghciwatch";
+    rev = "v${version}";
+    hash = "sha256-6afUHLPrSWhgN5LA346tAZ1+gROr+i/ZyCNVnyCd5Tc=";
+  };
+
+  cargoHash = "sha256-og7S3W+DCBlFIvKLZghLT+msBLnS1o7Rea7v2VPsDYA=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.CoreFoundation
+    darwin.apple_sdk.frameworks.CoreServices
+  ];
+
+  # integration tests are not run but the macros need this variable to be set
+  GHC_VERSIONS = "";
+  checkFlags = "--test \"unit\"";
+
+  meta = with lib; {
+    description = "Ghci-based file watching recompiler for Haskell development";
+    homepage = "https://github.com/MercuryTechnologies/ghciwatch";
+    license = licenses.mit;
+    maintainers = with maintainers; [ mangoiv _9999years ];
+    mainProgram = "ghciwatch";
+  };
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
i am fairly sure this does not go in meta, please verify.

Reply:
Did you mean "i am fairly sure this does not go in `passthru`"? Anyways I copied it from another package of mine, where it does work:

https://github.com/NixOS/nixpkgs/blob/8937b370727c9f8030a66449ab3ad0f9b5309069/pkgs/by-name/gi/git-upstream/package.nix#L30

```
$ rg -F passthru.updateScript | wc -l
1096
```

Reply:
I think i misread on account of reviewing on phone. it was below meta (unusual) and so i thought it was in meta.passthru.updateScript. can re-check when i get home